src/js/components/Chart/Chart.js

```diff
@@ -44,6 +43,8 @@ const Chart = React.forwardRef(
     const containerRef = useForwardedRef(ref);
     const { theme, passThemeFlag } = useThemeValue();
 
+    const thicknessValue = thickness || theme.chart?.thickness.default;
```

Comment:
Should we do `useMemo` here?

Reply:
ThicknessType is a simple atomic type (string or number) and the caculation here is minimal. I don't see any reason to memoize it here (ie not costly calculation and value won't cause a re-render in anything.)